devdoc/channel_internal.md

```diff
@@ -191,8 +226,10 @@ MOCKABLE_FUNCTION(, CHANNEL_RESULT, channel_internal_push, THANDLE(CHANNEL_INTER
 
 `execute_callbacks` is the work function that is passed to `threadpool_schedule_work` when scheduling the execution of the callbacks for an operation.
 
-**SRS_CHANNEL_INTERNAL_43_148: [** If `channel_internal_op_context` is `NULL`, `execute_callbacks` shall fail. **]**
+**SRS_CHANNEL_INTERNAL_43_148: [** If `channel_internal_op_context` is `NULL`, `execute_callbacks` shall terminate the process. **]**
 
 **SRS_CHANNEL_INTERNAL_43_145: [** `execute_callbacks` shall call the stored callback(s) with the `result` of the `operation`.  **]**
 
+**SRS_CHANNEL_INTERNAL_43_157: [** `execute_callbacks` shall call `sm_exec_end` for each call callback. **]**
```

Comment:
>for each call callback [](http://example.com/codeflow?start=80&length=22)

poetry.  Do you mean "each called callback"? Or maybe better phrased as "each callback that is called."

Reply:
Changed to "for each callback that is called"